packages/gatsby/content/getting-started/install.md

```diff
@@ -26,6 +26,12 @@ Corepack isn't included with Node.js in versions before the 16.10; to address th
 npm i -g corepack
 ```
 
+## Activate Yarn
+
+```bash
+corepack prepare yarn@stable --activate
```

Comment:
Note that this doesn't work in [`corepack@<0.12.0`](https://github.com/nodejs/corepack/releases/tag/v0.12.0)

Reply:
We've had a few reports of people running into that now so I reverted this PR from the live documentation, we need to be more specific.